src/components/inspectors/InspectorPanel.vue

```diff
@@ -198,17 +198,21 @@ export default {
 .inspector-container {
   text-align: left;
   user-select: none;
-}
 
-.form-accordtion-container {
-  padding: 0;
-}
+  .form-group {
+    padding: 0 0.5rem;
 
-.form-group {
-  padding: 0 0.5rem;
+    > label {
+      font-size: 0.8125rem;
+    }
+  }
+
+  .inspector-font-size {
+    font-size: 0.875rem;
+  }
 }
 
-.inspector-font-size {
-  font-size: 0.875rem;
+.form-accordtion-container {
```

Comment:
There's a typo in the word accordion here (accordtion). Is that correct?

Reply:
Not correct, thanks! It appears that there's no use of `form-accordtion-container` or `form-accordion-container` in modeler, form elements, screen builder, or processmaker, so I'm going to remove it.